sdks/python/scripts/run_pylint_2to3.sh

```diff
@@ -47,17 +47,23 @@ if test $# -gt 0; then
   esac
 fi
 
-FUTURIZE_GREP_PARAM=$( IFS='|'; echo "${ids[*]}" )
+EXCLUDED_FROM_FUTURIZE_CHECK=(
+# Non-portable code runs only in Python 2.
+# https://github.com/apache/beam/blob/7b8786f84739f4f50c2491cb247b75459734b5ac/sdks/python/apache_beam/typehints/typehints.py#L343
+"apache_beam/typehints/typehints.py"
```

Comment:
Could we just locally ignore the thrown pylint warning here, so we don't have to exclude the full module?

Reply:
It's not a pylint warning, but an output from futurize. Futurize produces the following output
```
RefactoringTool: Refactored apache_beam/typehints/typehints.py
--- apache_beam/typehints/typehints.py  (original)
+++ apache_beam/typehints/typehints.py  (refactored)
@@ -341,7 +341,7 @@
   # Must either be a TypeConstraint instance or a basic Python type.
   possible_classes = [type, TypeConstraint]
   if sys.version_info[0] == 2:
-    possible_classes.append(types.ClassType)
+    possible_classes.append(type)
   is_not_type_constraint = (
       not isinstance(type_param, tuple(possible_classes))
       and type_param is not None)
```
From which the script filters out the line ```RefactoringTool: Refactored apache_beam/typehints/typehints.py```, which does not have much details except for filename. We could add a directive in the code for ```futurize``` to ignore a line, but I did not see a way to do it - do we have one?